services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/design-kits.js

```diff
@@ -24,7 +37,18 @@ const DesignKits = ({ libraryData, navData }) => {
 
   const pageHeader = pageHeaders?.library ?? {}
 
-  const { name } = libraryData.content
+  const { name, designKits } = libraryData.content
+
+  const groupedDesignKits = groupBy(
```

Comment:
this is triggering an error for. me if I go on a library that doesn't have a `designKits` key (not empty array, but the key is undefined, which will happen if it wasn't specified in the carbon.yml):
<img width="1790" alt="image" src="https://user-images.githubusercontent.com/40550942/189401475-9030cad7-dace-420f-afa3-84ac5d0e5884.png">